tools/run_clippy.sh

```diff
@@ -178,6 +178,7 @@ CLIPPY_ARGS_PEDANTIC="
 -A clippy::verbose_bit_mask
 -A clippy::large_types_passed_by_value
 -A clippy::no_mangle_with_rust_abi
+-A clippy::struct-field-names
```

Comment:
I think this should be `clippy::struct_field_names` (the difference is `_` versus `-`).